stonesoup/models/measurement/tests/test_models.py

```diff
@@ -338,7 +338,77 @@ def fun(x):
                              ((0, 1, 0, 0, 0, 0), (10, 0, 0, 0, -10, 0),
                               (-np.pi / 4, 0, np.sqrt(200), -1 / np.sqrt(2))),
                              ((0, 0, 0, 0, 0, 1), (0, 0, 0, 0, 10, 0),
-                              (0, 0, 10, -1)),
+                              (0, 0, 10, -1)),    # original tests up to here
```

Comment:
There are quite a lot of flake8 errors in this block. (long lines and whitespace errors)

Reply:
My mistake - forgot to check with flake8. All fixed.